openshift-gitops/apps/charts/templates/cp.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   destination:
     namespace: {{ .Values.spec.destination.namespace.cp }}
-    server: {{ .Values.spec.destination.server.cp }}
```

Comment:
If these values are being modified in templates/cp.yaml, then it needs to be modified for all `templates/` files.

Reply:
I think you missed dp.yaml